### ISSUE TITLE
New Dynamic fonts rule

### ISSUE THREAD

**Issue by HonzaMac** [IBDecodable/IBLinter#143]
In our code base, we would like to encourage developers to use [Dynamic Fonts](https://developer.apple.com/documentation/uikit/uifont/scaling_fonts_automatically) (change size based on user preference).

This can be found in xib file as node:
`<fontDescription key="fontDescription" type="system" pointSize="15"/>`
can be replace by 
`<fontDescription key="fontDescription" style="UICTFontTextStyleSubhead"/>`

Basically, when xib contains `pointSize` we should encourage to use Defined dynamic font size instead.

Does it make sense for IBLinter?

**Comment by HonzaMac** [comment]
FYI:
There could be similar approach for custom swiftLint:
```
  dynamic_font_size:
    included: ".*.swift"
    name: "Prefer Dynamic fonts"
    message: "Please use dynamic preferred font styles instead of static system fonts"
    regex: '\.(italic|bold|monospaced|monospacedDigit)?[sS]ystemFont\('
    severity: warning
```

**Comment by HonzaMac** [comment]
I don't need it anymore.
